rollup.config.js

```diff
@@ -72,14 +72,9 @@ module.exports = [
     ],
   },
 
-  // CommonJS (for Node) and ES module (for bundlers) build.
-  // (We could have three entries in the configuration array
-  // instead of two, but it's quicker to generate multiple
-  // builds from a single configuration where possible, using
-  // an array for the `output` option, where we can specify
-  // `file` and `format` for each target)
+  // ES module (for bundlers) build.
   {
-    input: 'src/index.ts',
+    input: 'src/browser.ts',
     external: ['cross-fetch', 'cross-fetch/polyfill'],
```

Comment:
Is there a necessity to use both? or just the `'cross-fetch/polyfill'` is ok for this case?

Reply:
These are still compatible in `node`, with the only difference that the client will not have access to any method declared in the node-client!